toolbox

```diff
@@ -775,7 +775,7 @@ list_containers()
                       id=$(echo "$container" | cut --delimiter " " --fields 1 2>&3)
                       is_running=$($prefix_sudo podman inspect "$id" --format "{{.State.Running}}" 2>&3)
                       if $is_running; then
-                          echo -e "${LGC}$container${NC}"
+                          echo "${LGC}$container${NC}"
```

Comment:
We need to replace this `echo -e` with `printf` too.

(By the way, when I tested using `echo` without the `-e`, Bash refused to intepret the terminal escape sequences as control characters and printed them as is. So, maybe the behaviour that you saw was specific to Dash.)

Reply:
I took the liberty to make the above adjustments and merged to `master`. Thanks for your contribution!

Reply:
Thank you for merging!